backend/packages/Upgrade/src/api/repositories/LogRepository.ts

```diff
@@ -291,6 +291,7 @@ export class LogRepository extends Repository<Log> {
   ): SelectQueryBuilder<Experiment> {
     // get experiment repository
     const experimentRepo = getRepository(Experiment);
+    const nullstring = 'null';
```

Comment:
Is the null value in json is converted to string 'null'? So the question is what if someone uses undefined?